### PR TITLE
Added censor and fixed requirements

### PR DESCRIPTION
**Pull request by DWAA1660**
i dont know anything else to add to pr

### CODE REVIEW

File: secrets.env.template [hunk 1]
```diff
@@ -1,14 +0,0 @@
...
-# Used for weather command can be ignored in development
-OPEN_WEATHER_MAP_API_KEY=
-# Used for the upload checker can be ignored in development
-YT_API_KEY=
-POSTGRES_USER=
-POSTGRES_PASSWORD=
-POSTGRES_HOST=
-POSTGRES_PORT=
-# Needs to be set to true or false
-DEVELOPMENT=
```
Comment (OpenSourceSimon) [inline secrets.env.template:14]: Why did you remove this file?

File: cogs/AI.py [hunk 2]
```diff
@@ -13,7 +13,10 @@ def __init__(self, bot: OGIROID):
         self.bot = bot
 
     @commands.slash_command(description="Generates ai art")
-    async def ai_art(self, inter: disnake.ApplicationCommandInteraction, text):
+    async def ai_art(self, inter: disnake.ApplicationCommandInteraction, text: str):
+        if profanity.censor(text) != text:
+            return await inter.send(f"NSFW requests are not allowed!",
+                                    ephemeral=True)
```
Comment (OpenSourceSimon) [inline cogs/AI.py:19]: Interesting choice. Instead of this, you can also censor parts if it's a long command?

**Comment by LevaniVashadze** [comment]
Same question

File: secrets.env.template (hunk 1 above)
Comment (DWAA1660) [reply secrets.env.template]: accident i added now. you guys too fast

File: cogs/AI.py (hunk 2 above)
Comment (DWAA1660) [reply cogs/AI.py:19]: what do you mean this will just stop the command if any bad words are used it doesn't change prompt

Comment (OpenSourceSimon) [reply cogs/AI.py:19]: > what do you mean this will just stop the command if any bad words are used it doesn't change prompt

`profanity.censor(text)` This part censors the bad words. So you can just use that as the prompt maybe? If it's a really long prompt and there's only one word then it makes sense to just censor that one. But if the whole prompt is one word, then stop the command

**Comment by DWAA1660** [comment]
> Same question

as? who

**Comment by OpenSourceSimon** [comment]
> > Same question
> 
> as? who

As mine about the deleted .env file

**Comment by DWAA1660** [comment]
i added it back

**Review by LevaniVashadze — APPROVED** [review]
I approve

**Review by OpenSourceSimon — APPROVED** [review]
